Sources/MongoSwift/MongoClient.swift

```diff
@@ -89,18 +89,18 @@ public struct ClientOptions: CodingStrategyProvider, Decodable {
     }
 }
 
-/// Options to use when retrieving a `MongoDatabase` from a `MongoClient`.
+/// Options to use when retrieving a `MongoDatabase` from a `MongoClient`. If an option is not specified here, the
```

Comment:
similar to my comment above, I think the new text might make more sense to include in the docstring for `MongoClient.db`.